docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -100,34 +98,15 @@ The `fhir-search` module requires that the [expression](https://www.hl7.org/fhir
 
 A few things to note are: 
 - This SearchParameter includes an xpath element for completeness, but the IBM FHIR Server does not use the XPath during extraction; it only uses the expression (FHIRPath).
-- The SearchParameter with a path including `value` use the Choice data types which are determined based on the SearchParameter type and the Resource's data type (using the extension-search-parameters-valuetypes.json). 
+- The SearchParameter with a path including `value` use the Choice data types which are determined based on the SearchParameter type . 
 - Each time a resource is created or updated, the FHIR server evaluates the FHIRPath expression applicable to the resource type and indexes the values of the matching elements, making these available via a search where the query parameter name matches the `code` element on the `SearchParameter` definition.
 
 In the preceding example, extension elements (on a Patient resource) with a url of `http://ibm.com/fhir/extension/Patient/favorite-color` are indexed by the `favorite-color` search parameter. To search for Patients with a favorite color of "pink", users could send an HTTP GET request to a URL like `[base]/api/v1/Patient?favorite-color:exact=pink`.
 
 For more information on search parameters, see the HL7 FHIR specification.
 
-#### 1.1.2 Search Parameters Configuration: extension-search-parameters-valuetypes.json
-The extension-search-parameters-valuetypes.json must be configured in concert with each search parameter to identify the expected data type(s) for the element(s) selected by that search parameter's expression. 
-
-For each Search Parameter, there needs to be an entry with the following fields:
-    - `resourceType` - the FHIR resource type. 
-    - `name` - the SearchParameter code that matches to this mapping. 
-    - `targetClasses` - an array of fhir-model simple Class names that correspond to the expected data types for this search parameter. 
-
-```
-{
-    "value-types": "default-extensions",
-    "mappings": [{
-        "resourceType": "Patient",
-        "name": "favorite-color",
-        "targetClasses": ["String"]
-    }]
-}
-```
-
-#### 1.1.3 Test
-When creating a SearchParameter extension, it is recommended that the FHIRPath expression is checked for validity. Many examples exist in the module `fhir-search`. 
+#### 1.1.2 Test
+When creating a SearchParameter extension, it is recommended that the FHIRPath expression is checked for validity. Many examples exist in the module `fhir-search`. Further, optimizations to the custom SearchParameter extensions may add explict casts to the intended type and activate the specific behavior `refSequence.start-end.as(FHIRQuantity)`. An entry may look like `"expression": "Basic.extension.where(url='http://example.org/decimal').value.as(Decimal)"`.
```

Comment:
This line is confusing to me:
> Further, optimizations to the custom SearchParameter extensions may add explict casts to the intended type and activate the specific behavior `refSequence.start-end.as(FHIRQuantity)`.

Reply:
I also think we can be a little more clear that the server will skip any search parameters that cannot be mapped to the specified search parameter type.  And probably link to https://www.hl7.org/fhir/R4/search.html#table as the documentation for the allowed mappings.

Reply:
sure.  let's discuss post your first pass of the review

Reply:
since we are now removing the integer part, it's no longer needed above

Reply:
I've modified the paragaph

Reply:
> Further, optimizations to the custom SearchParameter extensions may add explict casts to the intended type and activate the specific behavior `refSequence.start-end.as(FHIRQuantity)`. An entry may look like `"expression": "Basic.extension.where(url='http://example.org/decimal').value.as(Decimal)"`.

Still not clear to me if we're recommending them to use `as` or saying the server might do it automatically?  Its the phrase "optimizations to the custom SearchParameter extensions may add explict casts" messing me up.
Maybe change that to:
> If a search parameter expression extracts an element with a data type that is incompatible with the declared search parameter type, the server will skip the value and log a message. For choice elements, like `Extension.value`, its recommended to restrict the expression to values of the desired type by using the `as` function. For example, select only Decimal values from the `http://example.org/demical` extension, use and expressions like `Basic.extension.where(url='http://example.org/decimal').value.as(Decimal)`.